website/pages/docs/theming/component-style.mdx

```diff
@@ -210,29 +216,38 @@ The computed styles for the component based on `props` passed. If no `props` is
 passed, the `defaultProps` defined in the style config will be used.
 
 ```jsx live=false
-// 1. Import useStyleConfig
-import { useStyleConfig } from "@chakra-ui/react"
+import { Box, useStyleConfig } from "@chakra-ui/react"
 
 function Card(props) {
-  const { variant, children, ...rest } = props;
-  const styles = useStyleConfig("Card", { variant });
+  const { variant, children, ...rest } = props
 
-  return (
-    <Box sx={styles} {...rest}>
-      {children}
-    </Box>
-  );
+  const styles = useStyleConfig("Card", { variant })
+
+  // Pass the computed styles into the `__css` prop
+  return <Box __css={styles} {...rest} />
```

Comment:
Hi @segunadebayo 
I read this piece of code in the documentation and wondered what happened to `children`. Shouldn't it read:

```suggestion
  return <Box __css={styles} {...rest}>{children}</Box>
```

Reply:
jsx allows `children` in a prop notation and as  _real_ children.

```jsx
<div children="Hello" /> === <div>Hello</div>
```

Spreading all remaining props equals to the example above.

Please note that _real_ children have precedence over the prop notation:
  
```jsx
<div children="Hello">World</div> === <div>World</div>
```

Reply:
Hey @TimKolberger 

thanks for your response. 
I agree that with the spreading of `{...rest}` on the component should pass `children` as props. 

However in line 222
```js
const { variant, children, ...rest } = props
```
`children` is destructed from `props` and is then not a part of `rest` object.

Reply:
@TimKolberger https://github.com/chakra-ui/chakra-ui-docs/pull/284 😉